src/TableOfContents/types.ts

```diff
@@ -0,0 +1,14 @@
+import { IconName } from '@blueprintjs/icons';
+import { NodeType } from '@stoplight/types';
+
+export interface IContentsNode {
+  name: string;
+  depth: number;
+  isActive?: boolean;
+  href?: string;
+  type?: 'divider' | 'group' | 'item';
+  icon?: IconName;
+}
+
+export type IconMapType = NodeType | 'group' | 'divider' | 'item';
```

Comment:
Can we remove NodeType from IconMayType? I don't see it being used in any  styling so it shouldn't be needed